cvat/apps/dataset_manager/task.py

```diff
@@ -36,6 +36,9 @@ def log_exception(logger=None, exc_info=True):
 def get_export_cache_dir(db_task):
     return osp.join(db_task.get_task_dirname(), 'export_cache')
 
+EXPORT_FORMAT_DATUMARO_PROJECT = "datumaro_project"
```

Comment:
```suggestion
EXPORT_FORMAT_DATUMARO_PROJECT = "Dataset repository"
```

Better to have a human readable name here. What do you think?

Reply:
It is not a name, visible names are declared somewhere in `js`  code.